### PR TITLE
Restores Travis config without the deployment step

### PR DESCRIPTION
**Pull request by sbastn**
Restores Travis build along with slack notifications

Screen Shot 2020-03-11 at 11 59 11 AM

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: ruby
+
```
Comment (weppos) [inline .travis.yml]: We need to pin to the same version we set in https://github.com/dnsimple/dnsimple-support/blob/master/.tool-versions#L1 and update the .tool-versions to inform to update this file as we do in other (private) apps.

Example:

```
# Update also: .travis.yml
ruby 2.6.5
```

Comment (jacegu) [reply .travis.yml]: +1

Comment (sbastn) [reply .travis.yml]: ✅ Done

**Comment by sbastn** [comment]
Verified that Travis is using the correct Ruby version https://github.com/dnsimple/dnsimple-support/pull/525/checks?check_run_id=521878258

Screen Shot 2020-03-20 at 1 54 55 PM

**Pull request merged**